src/ApiManagement/ApiManagement/ChangeLog.md

```diff
@@ -19,6 +19,51 @@
 -->
 ## Upcoming Release
 
+[Breaking change] Sample has been replaced by Examples in 
```

Comment:
Due to the length limit of the overall changelog of powershell modules, please make it consise.
I think we should keep the one about breaking changes, and summarize the rest with 1 sentence.